lib/src/theme.dart

```diff
@@ -271,7 +271,10 @@ ThemeData flutterNesTheme({
     markerSize: Size(24, 24),
     itemMinHeight: 32,
   ),
+  Iterable<ThemeExtension<dynamic>>? customExtensions,
 }) {
+  final extraExtensions = customExtensions ?? [];
+
```

Comment:
```suggestion
  Iterable<ThemeExtension<dynamic>> customExtensions = [],
}) {
```

And then use `customExtensions` instead of `extraExtrensions` a few lines below, wdyt?

Reply:
Sure thing! 

I'll work on this as soon as possible.